src/__init__.py

```diff
@@ -1,2 +1 @@
-from src import *
-from src.add import *
\ No newline at end of file
+from src.add import add
```

Comment:
you could also leave this file blank.
The default behaviour would be that you could elsewhere 
import src
or 
from src import add
or (possibly)
from src.add import add